spec/base_controller_spec.rb

```diff
@@ -1,6 +1,48 @@
 require "spec_helper"
 
 describe Neddinna::BaseController do
-  it "should have a request object" do
+  let(:request) { Rack::Request.new(Rack::MockRequest.env_for("/posts")) }
+  let(:obj) { PostsController.new(request) }
+  let(:post) { Post.create(author: "Bb", description: "render test",
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.